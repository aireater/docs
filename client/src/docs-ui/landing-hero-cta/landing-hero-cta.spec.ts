import {DocsLandingHeroCTA} from "./landing-hero-cta";
import {newSpecPage} from "@stencil/core/testing";

describe("docs-landing-hero-cta", () => {
  describe("Render logic", () => {
    it("should render", async () => {
      expect(
        (
          await newSpecPage({
            components: [DocsLandingHeroCTA],
            html: `<docs-landing-page-hero-cta />`,
          })
        ).root,
      ).toMatchSnapshot();
    });
  });
});
