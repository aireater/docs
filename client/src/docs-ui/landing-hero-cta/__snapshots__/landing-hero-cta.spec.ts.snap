// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`docs-landing-hero-cta Render logic should render 1`] = `MockHTMLElement {}`;
